blog/en-us/dubbo-basic-usage-dubbo-consumer-configuration.md

```diff
@@ -0,0 +1,193 @@
+# DubboBasic Usage - Dubbo Consumer Configuration
+
+## Dubbo Consumer Configuration
+
+### Consumer Configuration Detailed
+
+There are 3 ways to configure the Dubbo Consumer: XML configuration, API call mode configuration, and annotation mode configuration.
+
+#### XML Configuration
+
+###### Example of the simplest configuration:
+
+```
+
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    Xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    Xmlns:dubbo="http://dubbo.apache.org/schema/dubbo"
+    Xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-4.3.xsd http://dubbo.apache.org/schema /dubbo http://dubbo.apache.org/schema/dubbo/dubbo.xsd">
+    <dubbo:application name="hello-world-app" />
+    <dubbo:registry address="multicast://224.5.6.7:1234" />
+    <dubbo:protocol name="dubbo" port="20880" />
+    <dubbo:reference id="demoServiceRemote" interface="com.alibaba.dubbo.demo.DemoService" />
+</beans>
+```
+
+
+
+> For the supported configuration tags and corresponding configuration items, refer to the usage in the provider.
+
+> Next, focus on the configuration of &lt;dubbo:reference/&gt;.
+
+
+
+* &lt;dubbo:reference/&gt; List of main attributes supported:
+
+| Property Name | Description |
+| -------- | ----- |
+| id | service reference id, as java bean id, requires unique |
+Interface | interface name for finding services |
+| version | version number, consistent with the service provider's version |
+| timeout | service method call timeout (ms) |
+| retries | The number of retry attempts by the remote service, excluding the first call, no need to retry, please set to 0 |
```

Comment:
`no need to retry, please set to 0` is one sentence. considering use if no need to retry, please set to 0